client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/BackfillOperator.kt

```diff
@@ -82,7 +85,7 @@ internal class BackfillOperator<E : DbEntity<E>, Pkey : Any> internal constructo
   fun prepareBackfill(request: PrepareBackfillRequest): PrepareBackfillResponse {
     validateRange(request.range)
 
-    backfill.validate(BackfillConfig(request.parameters, request.dry_run))
+    backfill.validate(parametersOperator.getBackfillConfig(request))
```

Comment:
so effectively we're validating the params can deserialize here, nice

Reply:
Yep during the prepare. Also the prepare can use the parameters to do different things.